variables.tf

```diff
@@ -86,3 +86,15 @@ variable "description" {
   description = "A friendly description of the WebACL"
   default     = null
 }
+
+variable "enable_custom_response" {
+  type        = bool
+  description = "Set to `true` to define custom responses (status code, body, and/or headers) for non managed rule group statements block actions."
+  default     = false
+}
+
+variable "custom_response_body" {
+  type        = map(string)
+  description = "A custom response body to be referenced on a per rule basis."
+  default     = {}
+}
```

Comment:
Just say this PR! Looking to solve the same problem, WDYT about an array of custom response bodies so we are not limited to just one? Please see https://github.com/umotif-public/terraform-aws-waf-webaclv2/pull/57/files

Reply:
@naseemkullah have you tested that? I was not able to get it to work with an array input for `custom_response_body`.

Reply:
Just updated my PR with the inclusion of your purposed `custom_response_bodies`. 👍🏼  Good call.